handlers/service_type_handlers_test.go

```diff
@@ -270,7 +305,7 @@ func (suite *ServiceTypeHandlersSuite) TestServiceTypeCreateInvalidAuthMethod()
 
 	expRespJSON := `{
  "error": {
-  "message": "auth_method: unsup_method is not yet supported.Supported:[api-key x-api-token]",
+  "message": "auth_method: unsup_method is not yet supported.Supported:[api-key headers]",
```

Comment:
Maybe add a comma separating the different supported types: eg `[api-key, headers]` etc...